CONTRIBUTING.md

```diff
@@ -1,61 +1,78 @@
 # Filing an issue
 
 ### Must read
-* If you aren't sure, you can ask on the [**IRC channel**](https://webchat.freenode.net/?channels=qtox) or read our [**wiki**](https://github.com/tux3/qTox/wiki) first.
+* If you aren't sure, you can ask on the
+  [**IRC channel**](https://webchat.freenode.net/?channels=qtox) or read our
```

Comment:
I tried the change in `CONTRIBUTING` and `INSTALL`, it brings little gain. In most cases, either the URL alone is longer than the remaining columns, or the final amount of lines is the same breaking the `[]()` construct than leaving it together. I'd leave the remaining cases untouched for consistency.